builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -15,9 +16,16 @@ steps:
   inputs:
     version: '3.1.x'
 
-- script: npm install -g azure-functions-core-tools@3
+- script: npm install -g azure-functions-core-tools
   displayName: 'Install Azure Functions Core Tools'
 
+- script: npm install -g azurite
+  displayName: 'Install Azurite Local Storage Emulator'
+
+- bash: echo "##vso[task.setvariable variable=nodeModulesPath]C:\npm\prefix\node_modules"
```

Comment:
We generally shouldn't be hardcoding paths like this - especially when we don't control the machine. 

Where is this path coming from anyways? I'm not seeing this path on my machine...

Reply:
Didn't know of this command, will try with this

Reply:
Updated yaml to set the path variable from the output of "npm root -g"